### DISCUSSION TITLE
记得之前有个佬fork的一个版本可以设置系统角色指令什么的，图标是个黄色的机器人，请问有没有人知道仓库地址

### DISCUSSION THREAD

**Discussion by fangyuan99** [ddiu8081/chatgpt-demo#321] [Q&A]
![image](https://user-images.githubusercontent.com/62874116/227751500-c4e35677-ac0a-4956-93cd-a93638384bd3.png)
类似这样的

**Comment by ddiu8081** [comment]
https://github.com/ourongxing/chatgpt-vercel
